packages/common/src/core/types/IHubProject.ts

```diff
@@ -26,3 +27,16 @@ export enum PROJECT_STATUSES {
   onHold = "onHold",
   complete = "complete",
 }
+
+/**
+ * This type redefines the IHubProject interface in such a way
+ * that it can be consumed by the entity editor.
+ */
+export type IHubProjectEditor = Omit<IHubProject, "extent"> & {
+  extent: IExtent | number[][];
+  // TODO: if this is an ephemeral property, we should prefix with _
```

Comment:
I agree. We would just need to change the reference in the `uiSchema` to include an underscore as well.

Reply:
Done and verified in harness.